aws-cpp-sdk-cloudfront-integration-tests/RunTests.cpp

```diff
@@ -6,20 +6,26 @@
 #include <gtest/gtest.h>
 #include <aws/core/Aws.h>
 #include <aws/testing/platform/PlatformTesting.h>
+#include <aws/testing/TestingEnvironment.h>
 #include <aws/testing/MemoryTesting.h>
 
 int main(int argc, char** argv)
 {
+#ifdef __linux__
```

Comment:
you already include PlatformTesting.h / MemoryTesting.h
You can reduce code duplication by extracting logic required into one of this common headers and just declare there and call a method here called smth like "InstallSigPipeHandler()".

We should also consider making the handler more smart that tries to detect if it is a known SIGPIPE from curl from OpenSSL or something different that should not be ignored.

Reply:
New revision including this change.

Reply:
Thank you, ship it.